signed/verifiers.go

```diff
@@ -80,55 +81,34 @@ func (v RSAVerifier) Verify(key data.Key, sig []byte, msg []byte) error {
 		return ErrInvalid
 	}
 
-	if err = rsa.VerifyPKCS1v15(rsaPub, crypto.SHA256, digest[:], sig); err != nil {
+	opts := rsa.PSSOptions{SaltLength: sha256.Size, Hash: crypto.SHA256}
+	if err = rsa.VerifyPSS(rsaPub, crypto.SHA256, digest[:], sig, &opts); err != nil {
 		logrus.Infof("Failed verification: %s", err)
 		return ErrInvalid
 	}
 	return nil
 }
 
-type RSAPemVerifier struct{}
```

Comment:
This verifier and its Verify functions need to remain in their original format. They provide compatibility for the Python implementation of TUF.